docs/docs/getting-started/first-steps.md

```diff
@@ -64,7 +64,13 @@ If you encounter any problem with the following steps, make sure to use the [lat
 
     If you require the OS image to be available in another region, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md&title=Support+new+AWS+image+region:+xx-xxxx-x).
 
-    You can find a list of all [regions in AWS's documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+    You can find a list of all [regions in AWS' documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
```

Comment:
I think it was correct before https://www.google.com/search?q="aws's"+site%3Aamazon.com

Reply:
I feel like people at AWS aren't sure themself? :D
Taken from your search: 
exhibit A, both versions used: https://sustainability.aboutamazon.com/environment/the-cloud?energyType=true
exhibit B, both versions used: https://aws.amazon.com/security/

I am happy to change it back. My motivation for changing it: "With a plural noun ending in -s, you only need to add an apostrophe to the end of the word to make it possessive." 
https://www.britannica.com/dictionary/eb/qa/How-to-Make-Possessives-in-English

sidenote: while searching "aws possessive form" on google, I found a .docx document on possessive forms on a s3 bucket. unfortunately the document is not by AWS :D.

Reply:
Interesting that they don't use it uniformly. I remembered having seen it as "AWS's" and just did the quick Google check without further validation. Then I actually don't care.

Additional thoughts:
* I don't know if AWS is considered a plural noun. You think because the S means services? And if it would mean service, you'd write AWS's?
* chatgpt says: The possessive form of AWS (Amazon Web Services) would be "AWS's."

Reply:
Changed it back.